modules/data.atmosphere/R/extract_ERA5.R

```diff
@@ -21,85 +21,134 @@ ERA5_extract <-
   function(lat ,
            long ,
            years ,
-           vars = NULL, 
-           data.folder) {
-    
+           dbparms,
+           vars = NULL) {
+    # Distributing the job between whatever core is available. 
+     
+
     ensemblesN <- seq(1, 10)
     
+
     tryCatch({
-      # for each ensemble
-      one.year.out <- ensemblesN %>%
-        purrr::map(function(ens) {
+       #for each ensemble
+      one.year.out <- years %>%
+        purrr::map(function(year) {
+          
+        
+          tryCatch({
+            bety <- dplyr::src_postgres(dbname   = dbparms$dbname, 
```

Comment:
1) dbparms isn't documented as a function arg. 2) met workflow convention is to not create internal dependencies on the database as this prevents met functions from running remote.